src/modbus.h

```diff
@@ -68,6 +68,8 @@ MODBUS_BEGIN_DECLS
 #define MODBUS_FC_WRITE_MULTIPLE_COILS      0x0F
 #define MODBUS_FC_WRITE_MULTIPLE_REGISTERS  0x10
 #define MODBUS_FC_REPORT_SLAVE_ID           0x11
+#define MODBUS_FC_READ_GENERAL_REFERENCE 0x14
+#define MODBUS_FC_WRITE_GENERAL_REFERENCE 0x15
 #define MODBUS_FC_MASK_WRITE_REGISTER       0x16
```

Comment:
Same as above: I would prefer:
MODBUS_FC_READ_FILE_RECORD and MODBUS_FC_WRITE_FILE_RECORD; also looks like a whitespace problem...